src/profiler/Elastic.Apm.Profiler.Managed/Integrations/AdoNet/CommandExecuteNonQueryAsyncIntegration.cs

```diff
@@ -23,16 +23,14 @@ namespace Elastic.Apm.Profiler.Managed.Integrations.AdoNet
 	/// </summary>
 	[InstrumentMySqlAttribute(Method = ExecuteNonQueryAsync, ReturnType = TaskInt32, ParameterTypes = new[] { ClrTypeNames.CancellationToken })]
 	[InstrumentNpgsql(Method = ExecuteNonQueryAsync, ReturnType = TaskInt32, ParameterTypes = new[] { ClrTypeNames.CancellationToken })]
-	[InstrumentOracleManagedDataAccess(Method = ExecuteNonQueryAsync, ReturnType = TaskInt32,
-		ParameterTypes = new[] { ClrTypeNames.CancellationToken })]
-	[InstrumentOracleManagedDataAccessCore(Method = ExecuteNonQueryAsync, ReturnType = TaskInt32,
-		ParameterTypes = new[] { ClrTypeNames.CancellationToken })]
+	[InstrumentNpgsql(Method = ExecuteNonQuery, ReturnType = TaskInt32, ParameterTypes = new[] { ClrTypeNames.Bool, ClrTypeNames.CancellationToken })]

```

Comment:
Do we need to instrument this private method in addition to the public methods (async and sync) which call it?

Reply:
Great catch most likely not! I think this method is highly likely to be inlined too.